######################################################################
# GOAL
## docking waypoint heading to the charging station
bool start_autodock
---
######################################################################
# RESULT

## if docking succeed
bool is_success

## verbose description of the result
string status

---
######################################################################
# FEEDBACK

## progress approx, 0 - 1.0 
float32 progress

## verbose description of the progress
string status

## state
uint8 STATE_ERROR = 0
uint8 STATE_IDLE = 1
uint8 STATE_PREDOCK = 2
uint8 STATE_CORRECTION = 3
uint8 STATE_STEER_DOCK = 4
uint8 STATE_LAST_MILE = 5
uint8 STATE_SLIDER_GO_OUT = 6
uint8 STATE_SLIDER_GO_IN = 7
uint8 STATE_GO_OUT_DOCK = 8
uint8 STATE_ACTIVATE_CHARGER = 9
uint8 STATE_SUCCESS = 10
uint8 STATE_RETRY = 11
uint8 STATE_PAUSE = 12
uint8 STATE_CANCEL = 13

uint8 state
